docs/skopeo.1.md

```diff
@@ -245,6 +260,9 @@ $ skopeo standalone-verify busybox-manifest.json registry.example.com/example/bu
 Signature verified, digest sha256:20bf21ed457b390829cdbeec8795a7bea1626991fda603e0d01b4e7f60427e55
 ```
 
+# SEE ALSO
+kpod-login(1)
```

Comment:
Also add docker-login(1)

Reply:
Should we not this in the Buildah/kpod commands too?  I'm torn, we do use docker-login for those too, just don't want people to think we didn't think about that if we don't document it.  Then again I'd rather not add a docker dependency to them.  i.e. if docker changes auth at some point, then we'd have to adjust those for that rather than just dropping it.

Reply:
fixed.
@rhatdan should this be mentioned in buildah and kpod as well?

Reply:
We can mention in buildah and kpod that docker login works, but don't need to specify it directly.  Since those tools are not as tied into the Docker infrastructure as skopeo is.